### DISCUSSION TITLE
Github Copilot Student Access

### DISCUSSION THREAD

**Discussion by mursalfk** [community/community#23104] [New to GitHub]
<p>I am a student, but I didn’t get free Github Copilot access. How to resolve this?</p>

**Comment by jfullstackdev** [comment]
<p>try asking it here</p>
<p><a href="https://github.com/orgs/github-community/discussions/categories/github-education">https://github.com/orgs/github-community/discussions/categories/github-education</a></p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Howdy :cowboy_hat_face:</p>
<p>Have you checked out the thread <a class="mention" href="/yistarostin">@yistarostin</a> posted <a href="https://github.community/t/how-can-students-get-access-to-copilot-through-student-pack/257736/3">here</a>?</p>

**Comment by matteocontrini** [comment]
<p>You can enable it here: <a href="https://github.com/github-copilot/free_signup" class="inline-onebox" rel="noopener nofollow ugc">Sign in to GitHub · GitHub</a></p>
